backend/pubspec.yaml

```
@@ -28,3 +28,4 @@ dev_dependencies:
   json_serializable: ^6.8.0
   mocktail: ^1.0.3
   test: ^1.25.5
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)